### ISSUE TITLE
dompdf table is displaying wrong after pagebreak

### ISSUE THREAD

**Issue by kanapes** [dompdf/dompdf#2676]
We are using PHP Version 7.3.31 together with the latest stable domPDF version 1.1.1.

After rendering html into pdf dompdf fil up the text 3. + 4. column to the left.
It looks like that dompdf is loosing the table information after pagebreak during the rendering process

Maybe it will help to have a look into the extended file. Please have a look on the generated pdf file and scroll to the error on page 8 ("test.php")

[test.zip](https://github.com/dompdf/dompdf/files/7659769/test.zip)

don't be confused cause i changed the letter, character and digits in the html base file to 'xxxxx'
Style and class attributes are removed too. Hope this helps !?

regards###

**Comment by chepe12** [comment]
I have the same problem if you resolve please share the information

**Comment by bsweeney** [comment]
Where things break is a bit hard to follow precisely because everything is all x's. It would help if you could simplify the test case.

That being said I believe you're running into issue #1388.

**Comment by kanapes** [comment]
Thanks Brian - yes issue #1388 looks like my problem. 

Have not seen that issue by using the searchengine, sorry for that.
I believe issue #1388 describes my problem very well but if you still need more test cases i can modifie my files. 

Just let me know 

By the way, the start post at issue #1388 makes me a little worried if i have a look on the date.
I do not want to push there is hope for a solution soon?

**Comment by bsweeney** [comment]
Since you've identified 1388 as the likely cause no need to simplify your example. When a fix is ready I'll most likely run this example through just to check how it renders.
